test/eventsource.h

```diff
@@ -24,6 +24,12 @@
 #include <rapidjson/document.h>
 #include <vector>
 
+#if defined(__FreeBSD__)
+#include <unistd.h>
+typedef u_long ulong;
+using namespace std;
```

Comment:
I don't understand why this is needed?

Reply:
This is needed for the test pass. FreeBSD has ulong defined in <sys/unistd.h> and for the test to pass, it looks for u_long in the code which isn’t a thing on the system.